Create 2022-08-12-unlocking-tssp-accounts.md (#119)
DFE-Digital/tra-digital-design-history: app/posts/find-a-lost-trn/2022-08-12-unlocking-tssp-accounts.md

---
  title: Unlocking Teacher Self-Service Portal accounts
  date: 2022-08-12
---

Many people use Find a Lost TRN to access their DQT record via the [Teacher Self Service Portal](). 

TSSP accounts that were created before 2014 and haven't been used are locked by default. At present, the only way to unlock your account is to contact the TRA helpdesk. 

This means that many of our users still have to contact the helpdesk after retrieving their TRN to finish their task.

## Automatic account unlocking

When we have matched someone to a TRN, we will set their record to unlocked automatically if:

> - it is locked,

Reviewer: ```suggestion
- it is locked
```

- it doesn't have any alerts (we can't unlock records with alerts relating to prohibitions, because people who are prohibited have their TRN published to GOV.UK, so anyone could access their record through the portal), and

> - the email address the user gave us hasn't been used in the past to find any other TRN.

Reviewer: ```suggestion
- the email address given has not been used to find another TRN
```
